www/client.js

```diff
@@ -9,13 +9,18 @@ let app = new Vue({
         hippoName: null,
         score: null,
         isPlaying: true,
+        noseGoes: {
+            isActive: false,
+            marbleX: 0,
+            marbleY: 0,
+        },
     },
 
     methods: {
         feedMe: function () {
             // If the user taps after they've lost, don't do anything.
             // TODO: Can we have Vue remove the binding when `isPlaying` is false?
-            if (!this.isPlaying) {
+            if (!this.isPlaying || this.noseGoes.isActive) {
```

Comment:
They can't eat until they tap the poison pill, but once they do they can go back to scoring points, even if the nose-goes event is active. I was actually thinking of changing it and making it so that nobody can score points until the nose-goes event fully ends.

I think we can discuss how we want this to behave, though. We have room to iterate on how the nose-goes event works.